### ISSUE TITLE
Post-Doc in Deep Learning for Climate Change [Milan, Italy]

### ISSUE THREAD

**Issue by Djeutsch** [Djeutsch/ldjeutchouang#20]
The Post-Doc positions is in Deep Learning for climate change impacts/mitigation, with a particular emphasis on convolutional neural networks (CNNs) application.

**Comment by Djeutsch** [comment]
## **Qualifications:**

- Phd in computer science, applied mathematics, computational economics
- Proficiency in Python programming with hands-on experience in Keras.
- Familiarity with PyTorch or TensorFlow is highly valued.
- User-level proficiency in bash scripting and UNIX environments, especially for supercomputing applications.
- Experience with Dask is a distinct advantage.
- Interest and ideally some experience in climate change research

## **Role Overview:**

You will be at the forefront of applying deep learning to address environmental and societal challenges. A particular emphasis will be on convolutional neural networks (CNNs) application to climate change physical and economic risks or on climate mitigation action.

## **Key Responsibilities:**

Design and implement a comprehensive deep learning architecture with a global perspective, integrating diverse spatial and temporal resolutions with the aim of uncovering the determinants of climate change impacts and progress on emissions reduction efforts. 

## **Modalities:** 

The researcher will join the ERC project [EUNICE ](https://www.eunice-erc.eu/)(https://www.eunice-erc.eu/) under the supervision of Professor Massimo Tavoni at Politecnico di Milano, a leading technical university. The project aims to quantify the deep uncertainties associated with climate mitigation and adaptation strategies, advancing the foundations of mathematical modeling and data science for climate change policy. The candidate will join a group composed by a climate economist, a climate scientist and three data scientists. The researcher will have the opportunity to be affiliated with the European Institute on Economics and the Environment (https://www.eiee.org/). 

Post-doc positions will last for 2 years with the possibility of renewal. Place of work is Milan, Italy, with some flexibility in smart working. Salary is competitive.

Interested candidates should send their CV to [massimo.tavoni@polimi.it](mailto:massimo.tavoni@polimi.it)
